### PR TITLE
Removing more AppBundle

### PR DESCRIPTION
**Pull request by weaverryan**
This is still not all of them (and we have a few pending PR's which remove some additional ones). But this gets us *close* to all AppBundle gone.

I've even renamed some instances that are not technically problematic. That's because I want to be able to do a `git grep AppBundle` on 4.0 and higher and see *nothing*. This will make it easier to spot merge issues later.

This helps #7809.

### CODE REVIEW

File: doctrine/multiple_entity_managers.rst [hunk 1]
```diff
@@ -50,12 +50,21 @@ The following configuration code shows how you can configure two entity managers
...
+                                alias: App
                     customer:
                         connection: customer
                         mappings:
-                            AcmeCustomerBundle: ~
+                            AppCustomer:
+                                is_bundle: false
+                                type: annotation
+                                dir: '%kernel.project_dir%/src/CustomerEntity'
+                                prefix: 'App\Customer\Entity'
```
Comment (yceruto) [inline doctrine/multiple_entity_managers.rst]: I wonder if it's part of the entity namespace shouldn't be `dir: '%kernel.project_dir%/src/Customer/Entity'`? or vice-versa `prefix: 'App\CustomerEntity'`?

Comment (ricknox) [reply doctrine/multiple_entity_managers.rst]: I rather think that it should be `%kernel.project_dir%/src/Entity/Customer/`.

In full:

```yaml
AppCustomer:
    is_bundle: false
    type: annotation
    dir: '%kernel.project_dir%/src/Entity/Customer/'
    prefix: 'App\Entity\Customer'
```

**Comment by weaverryan** [comment]
@yceruto @ricknox You guys were right - the example wasn't correct... and also wasn't a very good example. Can you look again? I put the entities for each EM into subdirectories, because I think if we use `src/Entity` for one EM and `src/Entity/Customer` for the second EM, the first EM will *also* map the entities in `src/Entity/Customer`, which we obviously do not want :)

**Review by ghost — APPROVED** [review]
@weaverryan This looks imo. much better then the previous example and makes more sense to me.

**Review by yceruto — APPROVED** [review]

**Pull request closed without merging**